src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -68,6 +73,8 @@ const BG_COLORS_BY_DARK_MODE_AND_CHAIN_ID: {
       'radial-gradient(182.71% 205.59% at 2.81% 7.69%, rgba(130, 71, 229, 0.2) 0%, rgba(167, 202, 255, 0.2) 100%)',
     [SupportedChainId.POLYGON_MUMBAI]:
       'radial-gradient(182.71% 205.59% at 2.81% 7.69%, rgba(130, 71, 229, 0.2) 0%, rgba(167, 202, 255, 0.2) 100%)',
+    [SupportedChainId.CELO]: 'rgba(50, 200, 167, 0.15)',
+    [SupportedChainId.CELO_ALFAJORES]: 'rgba(50, 200, 167, 0.15)',
```

Comment:
@ppphhhiiilll dark mode--same q.

Reply:
For reference

![Screen Shot 2022-06-28 at 1 42 09 PM](https://user-images.githubusercontent.com/31524583/176248181-2b954232-1b0e-4c55-bd77-7108e88d733c.png)

Reply:
can you try this instead: `radial-gradient(150% 100% at 50% 0%, rgb(2 80 47) 2%, rgb(12 41 28) 53%, rgb(31, 33, 40) 100%)`?

Reply:
Plot twist, I used that rident for the dark mode background color (as seen below) I think the network alert banner needs to be a bit brighter. WDYT @JFrankfurt ? 

<img width="1343" alt="Screen Shot 2022-07-01 at 9 39 10 AM" src="https://user-images.githubusercontent.com/31524583/176905887-76450c0b-0c9b-43a3-90b5-59b8dd46cc41.png">

Reply:
@ppphhhiiilll

Reply:
![image](https://user-images.githubusercontent.com/5773490/177380248-c7f2e856-b4ea-4bfa-a295-10a25ebe5b57.png)

Reply:
Alright - I updated it, let me know what you think. Happy to iterate in order to get this right!